firestore.rules

```diff
@@ -61,6 +61,12 @@ service cloud.firestore {
         allow update: if (request.auth.uid in role('admin').users || request.auth.uid in venueData().owners) &&
           isUpdateRestrictedToField(request, 'deleted');
       }
+      match /access/{method} {
+        allow read, write: if false;
+      }
+      match /accessgranted/{userId} {
+        allow read, write: if false;
+      }
```

Comment:
This seems like it's a placeholder implementation? Or is it's intent to always deny access on the frontend, such that it can only be read/manipulated via the backend functions?

Reply:
I think this is supposed to always deny access on the frontend.

Reply:
Makes sense